docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.3"
+services:
+  near-node:
+    build: .
+    network_mode: host
+    command: bash -c "node index.js start near-node; node index.js start near-node && while true; do sleep 30; done;"
+  ganache:
+    build: .
+    network_mode: host
+    command: >
+      bash -c "
+        while ! nc -z localhost 3030; do
+          sleep 1
+        done
+        node index.js start ganache --daemon false
```

Comment:
This issue unfortunately breaks it: https://github.com/near/rainbow-bridge/issues/96

Reply:
As commented on: https://github.com/near/rainbow-bridge/issues/96. Adding the command as is hardcoded for now with an installation of npm -g install ganache-cli in the image
```
ganache-cli --port 9545 --blockTime 12 --gasLimit 10000000 --account='0x2bdd21761a483f71054e14    f5b827213567971c676928d9a1808cbfa4b7501200,10000000000000000000000000000'
```